packages/text-input-react/src/TextArea.tsx

```diff
@@ -8,11 +8,15 @@ interface Props {
     isInvalid?: boolean;
     id?: string;
     required?: boolean;
+    helpText?: string;
+    errorText?: string;
 }
 
-export const TextArea = ({ isInvalid = false, id, label, className = "", ...rest }: Props) => (
+export const TextArea = ({ isInvalid = false, id, label, className = "", helpText, errorText, ...rest }: Props) => (
     <label data-testid="jkl-text-field" className={`jkl-text-field jkl-text-area ${className}`}>
         <textarea aria-invalid={isInvalid} className="jkl-text-field__input" id={id} placeholder=" " {...rest} />
         <span className="jkl-text-field__label">{label}</span>
+        {helpText && <span className="jkl__text jkl__text--help">{helpText}</span>}
```

Comment:
Help text should be replaced by error text, so we need `&& !(isInvalid && errorText)` here

Reply:
i fix